setup.py

```diff
@@ -17,14 +22,14 @@ def get_version():
     """
     locals_dict = {}
     globals_dict = {}
-    with open(os.path.join('nilearn', 'version.py')) as fp:
+    with open(os.path.join(__curdir__, 'nilearn', 'version.py')) as fp:
```

Comment:
Out of interest, what do you gain by being able to execute setup.py outside of the directory where it lives?